### ISSUE TITLE
Resources and Support content request for Vet Centers (submitted by Sitewide - Facilities)

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#65122]
## About your team

- Team name: Sitewide-Facilities
- OCTO-DE product owner: Michelle Middaugh
- Product manager: Jane Xiong
- Designer: Jordan Wood and Dave Pickett
- Accessibility specialist: Laura Flannery
- Content writer: 
- Slack channel: #sitewide-facilities
 
### Requesting draft of Resources and Support content to replace existing Vet Center legacy pages as discussed with RCS leadership on 9/5/23

Please communicate/coordinate with Jessica Schieffer (Jessica.Schiefer@va.gov) and Barb Kuhn (barbara.kuhn@va.gov)

</br>
</br>

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [ ] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [x] Collaboration on existing content 

#### IA
- [ ] User Flows
- [ ] Page placement
- [x] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No
  
## Supporting artifacts

- [#59933 [Epic] Vet Center CLP](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/59933)
- [Existing Vet Center pages](https://vetcenter.va.gov/VETCENTER/index.asp)
- [Example Vet Center facility page](https://www.bing.com/ck/a?!&&p=7a885425789260e9JmltdHM9MTY4NjA5NjAwMCZpZ3VpZD0zOGFlODk4OS0yZTkzLTZkZTItMjdhNC05YWExMmYzYjZjZjQmaW5zaWQ9NTIyNQ&ptn=3&hsh=3&fclid=38ae8989-2e93-6de2-27a4-9aa12f3b6cf4&psq=reno+vet+center&u=a1aHR0cHM6Ly93d3cudmEuZ292L3Jlbm8tdmV0LWNlbnRlci8&ntb=1) - these pages have FAQs at the bottom which contain some additional content
- [Vet Center CLP Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1685481128812/f1ffcfe6660189e8e0a0202267cc978786d43e2f?wid=0-1686161588018&outline=open)
- [Vet Center content strategy.docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcVI1IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b043cfc267f4b05d8ff461dbfea3e4898da3c5d1/Vet%20Center%20content%20strategy.docx)

**Comment by jennymayoco** [comment]
Hi @mmiddaugh ! Thank you submitting this request! Is there specific URL feedback you are looking for for the R&S page?

**Comment by mmiddaugh** [comment]
@jennymayoco I think @laurwill may have thoughts on the URL

**Comment by kristinoletmuskat** [comment]
Confirmed with Laura that we don't need IA specific work here, so untagging us.

**Comment by laurwill** [comment]
Hi @mmiddaugh , just confirming that I sent Jessica and Barbara the page content for review — I cc'd you on the email, and we'll update here when we get their feedback. Thank you!

**Comment by mmiddaugh** [comment]
@laurwill 
Barb and I spoke earlier today on Facilities topics and she asked me to share this brochure with you for language which can be leveraged to bring in the Vet Center personality/vibe.  She and Jessica can provide the specifics.  [VC_Brochure_Summary_V3_9.2023.pdf](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMHRXQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ad80df0213e4b17dcd87d22f0b49bfbf3bb98032/VC_Brochure_Summary_V3_9.2023.pdf)

**Comment by laurwill** [comment]
Thanks @mmiddaugh ! So they're planning to send specific feedback to the page draft?

This is an interesting content project because we want most of the core content (FAQs, etc) to be reused across the R&S page and the campaign landing page. But the Vet Center voice and tone doesn't completely match the VA.gov voice and tone, so this may be a case where they can leverage other parts of the campaign landing page for the more promotional language. Happy to chat more about this!

**Comment by jennymayoco** [comment]
Untagging myself and will track new IA work that comes in through those child tickets
